sdk/src/impl/whirlpool-impl.ts

```diff
@@ -328,65 +335,117 @@ export class WhirlpoolImpl implements Whirlpool {
     destinationWallet: PublicKey,
     positionWallet: PublicKey,
     payerKey: PublicKey
-  ): Promise<TransactionBuilder> {
+  ): Promise<{ ataTx?: TransactionBuilder; closeTx: TransactionBuilder }> {
```

Comment:
Can we return TransactionBuilder[] instead? 
I'm going to add a helper function in common-sdk to help send multiple transactions:
https://github.com/orca-so/orca/commit/642f43b22d67cec5232a3bf1e374f9246adcb80f#diff-761e6344537f10bbe5276e42c950aee2734c51cf3505d7ec6e2d23273406f5f2L235

Moving to an array also gives us the flexibility of adding transaction-packing to this method in the future (ex. make sure txn is full before moving on to the next transaction).